lib/sleet/branch.rb

```diff
@@ -22,7 +22,8 @@ def builds_with_artificats
     attr_reader :github_user, :github_repo, :branch, :circle_ci_token
 
     def url
-      "https://circleci.com/api/v1.1/project/github/#{github_user}/#{github_repo}/tree/#{branch}?filter=completed"
+      "https://circleci.com/api/v1.1/project/github/#{github_user}/#{github_repo}/tree/#{branch}" \
+        '?filter=completed&limit=100'
```

Comment:
Weird that I decided to just throw query params in the URL lol
Always fun to see the decisions you made a few years ago!

Like you breaking the query params to a new line!